ts/modules/drilldown.src.ts

```diff
@@ -1276,39 +1276,45 @@ if (PieSeries) {
                     (this.chart.drilldownLevels as any).length - 1
                 ],
                 animationOptions =
-                    (this.chart.options.drilldown as any).animation,
-                animateFrom = level.shapeArgs,
-                start = (animateFrom as any).start,
-                angle = (animateFrom as any).end - start,
-                startAngle = angle / this.points.length,
-                styledMode = this.chart.styledMode;
-
-            if (!init) {
-                this.points.forEach(function (
-                    point: Highcharts.PiePoint,
-                    i: number
-                ): void {
-                    var animateTo = point.shapeArgs;
-
-                    if (!styledMode) {
-                        (animateFrom as any).fill = level.color;
-                        (animateTo as any).fill = point.color;
-                    }
+                    (this.chart.options.drilldown as any).animation;
+
+            // Unable to drill down in the horizontal item series #13372
+            if (this.type !== 'item' && this.center) {
```

Comment:
Don't do hard checks on `this.type`, as it won't work for future series types that build upon the item series. Instead, use `this.is('item')`, or preferably do a feature check where it actually goes wrong.